src/types.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * subset of node.js types, safe to use in browser and bundlers
+ */
+declare const process: { env: { NODE_ENV?: string } };
+declare const console: { warn: (...args: Array<any>) => void };
```

Comment:
Since we do not load neither `@types/node` or `dom` in our code, need to provide shims for this API.

(Fun fact, `console` object is not a part of Javascript spec, it is separately described in Node and DOM specs)